docs/CONFIG-VARS.md

```diff
@@ -104,7 +104,7 @@ When setting V4_CFG_MANAGE_STORAGE to true, A new storage classes will be create
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
 | V4_CFG_ORDER_NUMBER | SAS order number | string | | true | | viya |
 | V4_CFG_CADENCE_NAME | Cadence name | string | lts | false | [stable,lts] | viya |
-| V4_CFG_CADENCE_VERSION | Cadence version | string | 2020.1 | true | | viya |
+| V4_CFG_CADENCE_VERSION | Cadence version | string | "2020.1" | true | Please quote the string to specify the version correctly | viya |
```

Comment:
For the note field:

```text
This value must be quoted to accommodate the updated SAS Cadence Version format. If the value is not quoted the deployment will fail.
```

Reply:
Update the note with provided sample.

Reply:
Did you actually mean to put a single quote in there, sorry little confused by your suggestion?
> The string must be surrounded by quotation marks`"` to accommodate the updated SAS Cadence Version format.

Reply:
Now I look at it, maybe we should say double quotation marks.

Reply:
It doesn't matter either is fine, I guess specifying double will just add more questions as why not single.

Reply:
Final version:
> This value must be surrounded by quotation marks to accommodate the updated SAS Cadence Version format. If the value is not quoted the deployment will fail.